keyboards/ropro/info.json

```
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "ropro",
+  "url": "https://github.com/TheRoyalSweatshirt/The_RoPro",
+  "maintainer": "[Garret G.](https://github.com/TheRoyalSweatshirt)",
+  "width": 13,
+  "height": 6,
+  "layouts": {
+    "LAYOUT_ortho_6x13": {
```

This needs to match the name used in `ropro.h`.

```suggestion
    "LAYOUT": {
```